### PR TITLE
Fix/slow site

### PR DESCRIPTION
**Pull request by ShubhamCanMakeCommit**
## Description

> Spliting `app.js` bundle for better performance and caching. 

## Changes

- Moved `frontend/public/index.html` to `frontend/src/index.html`
- Changed webpack config files and server.js

## Issue

> #450 

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- You can check in developer menu. The size should have decreased.

### CODE REVIEW

**Comment by ShubhamCanMakeCommit** [comment]
@alexanmtz please test with the latest changes. This will improve the performance a lot.

File: frontend/package.json [hunk 1]
```diff
@@ -40,16 +40,20 @@
...
     "enzyme-adapter-react-16": "^1.2.0",
     "enzyme-react-intl": "^2.0.2",
     "extract-react-intl": "^0.6.0",
     "extract-react-intl-messages": "^0.10.0",
     "fetch-mock": "^7.0.5",
+    "html-webpack-plugin": "^4.0.1",
     "jest": "^23.5.0",
     "jest-cli": "^23.6.0",
     "jest-webpack": "^0.5.1",
+    "mini-css-extract-plugin": "^0.9.0",
```
Comment (alexanmtz) [inline frontend/package.json]: When we try to deploy, Heroku logs the following error:
`Cannot find module 'mini-css-extract-plugin'`

So I think we cannot move here, cause Heroku use this lib in production.

Comment (ShubhamCanMakeCommit) [reply frontend/package.json]: I have made the changes. I think your'e correct. Heroku doesn't install dev dependecies that is why all the plugins in our project our installed in package.json

**Comment by ShubhamCanMakeCommit** [comment]
It just loaded like a zap.!! I think heroku also makes some optimization. 
Total Under 700KB.
Material UI under 300KB.
image

**Comment by ShubhamCanMakeCommit** [comment]
Slack Icon now loads. Commit `replaced back`.
Total Still Under 900KB.

**Comment by alexanmtz** [comment]
@SavvyShah, you have a bounty to claim from a long time: https://gitpay.me/#/task/316

Please submit your PR for the Send solution and claim your bounty.

**Pull request merged**